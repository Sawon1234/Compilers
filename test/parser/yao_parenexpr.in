yao := ((3) + (2));