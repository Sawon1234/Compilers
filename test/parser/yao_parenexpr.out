yao := 3 . plus < > ( 2 ) ;